kratos/utilities/coordinate_transformation_utilities.h

```diff
@@ -1052,6 +987,117 @@ class CoordinateTransformationUtils {
 	///@name Private Operations
 	///@{
 
+    void RotateRHSAux(TLocalVectorType& rLocalVector,
+                      GeometryType& rGeometry, const bool revert_rotation) const
```

Comment:
```suggestion
    void RotateRHSAux(
        TLocalVectorType& rLocalVector,
        GeometryType& rGeometry,
        const bool RevertRotation) const
```
Please follow the [style guide](https://kratosmultiphysics.github.io/Kratos/pages/Kratos/For_Users/Conventions/Style_Guide.html) in new code.

Reply:
Documentation is also missing.